### DISCUSSION TITLE
Behavior change on "/" character encoding

### DISCUSSION THREAD

**Discussion by PaulArnaud** [encode/httpx#2790] [Potential Issue] [answered]
Hello, I have just made the upgrade from 0.23.3 to 0.24.1 and I noticed a behavior change on the way to encode the parameters.

Here a small example to show the change:
```
import httpx

url = httpx.URL("http://example", params={"timezone": "Europe/Zurich"})
print(str(url))
```
The code with httpx version 0.23.3 produces this output
`http://example?timezone=Europe%2FZurich`

The code with httpx version 0.24.3 produces this output (same with version 0.24.0)
`http://example?timezone=Europe/Zurich`

Do I miss something ? 
Is the good practice to encode my parameters before passing them to httpx ?

**Comment by lovelydinosaur** [comment, marked as answer]
Resolved in https://github.com/encode/httpx/pull/2723 (not yet released)

It looks like it needs adding to the CHANGELOG tho.

If you track back through the issues you'll see the history. Both `%2F` and `/` are valid there, and browsers use `/`. But we've come to the conclusion that we probably do want to stick with `%2F` as the more robust variant.
